File: CompParalela/Trabalho Prático 2 - CRUD/TP2_Server/src/com/condelar/persistencia/Conexao.java
package com.condelar.persistencia;

import java.sql.Connection;
import java.sql.DriverManager;
import java.sql.SQLException;

public class Conexao {
	private String banco;
	private String host;
	private String usuario;
	private String senha;

	public Conexao() {
		this.banco = "tp2";
		this.host = "localhost";
		this.usuario = "root";
		this.senha = "35741458";
	}

	public Connection getConexao() throws SQLException {
		String url = "jdbc:mysql://" + this.host + "/" + this.banco;
		return DriverManager.getConnection(url, this.usuario, this.senha);
	}
}

File: 6º Período/Programação para Internet/21 - Revisão/conexao.php
<?php 

	$servidor="localhost";
	$usuario="root";
	$senha="";
	$banco="aularevisao";

	$conector=new mysqli($servidor,$usuario,$senha,$banco);

	if(mysqli_connect_errno())
	{
			trigger_error(mysqli_connect_errno());
	}
	
 ?>